prober/prober.sh

```diff
@@ -0,0 +1,11 @@
+set -eEuo pipefail
+
+ID_TOKEN=$(curl "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity?audience=$AUDIENCE" -H "Metadata-Flavor: Google")
```

Comment:
```suggestion
ID_TOKEN=$(curl -sF "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity?audience=$AUDIENCE" -H "Metadata-Flavor: Google")
```

Reply:
Talked with Seth, made sure it's actually '-sf'. Fixed now.